internal/service/plugin.go

```diff
@@ -129,8 +191,17 @@ func (p *PluginEmitter) Run() {
 							next++
 							nextStr = fmt.Sprint(next)
 							wait.Wait()
+							currentBlock++
+							e := p.dao.SaveProcessedBlockNum(context.TODO(), int(currentBlock))
```

Comment:
Do we need to save this in both the redis and the DB? Does other logic depend on it being in redis?